lib/katello/plugin.rb

```diff
@@ -123,7 +123,7 @@
        :content_hosts,
        :caption => N_('Content Hosts'),
        :url => '/content_hosts',
-       :url_hash => {:controller => 'katello/api/v2/systems',
+       :url_hash => {:controller => 'api/v2/hosts',
```

Comment:
It lives in the foreman controllers.  One of The goals of host unification
is to migrate to as much of the host object as possible.
On Feb 15, 2016 10:15 AM, "Chris Peters" <notifications@github.com> wrote:

> In lib/katello/plugin.rb
> <https://github.com/Katello/katello/pull/5764#discussion_r52909845>:
>
> > @@ -123,7 +123,7 @@
> >         :content_hosts,
> >         :caption => N_('Content Hosts'),
> >         :url => '/content_hosts',
> > -       :url_hash => {:controller => 'katello/api/v2/systems',
> > +       :url_hash => {:controller => 'api/v2/hosts',
>
> @jlsherrill <https://github.com/jlsherrill> Out of curiosity; where does
> this controller live? All others are prefixed with katello. I did some
> poking around and couldn't find anything.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello/pull/5764/files#r52909845>.
>